src/commands/fun/meme.ts

```diff
@@ -0,0 +1,27 @@
+import fetch from 'node-fetch';
+import Command from '../../lib/structures/Command';
+import { TypicalGuildMessage } from '../../lib/types/typicalbot';
+import { MessageEmbed } from 'discord.js';
+
+export default class extends Command {
+    async execute(message: TypicalGuildMessage) {
+        const { data: { children } } = await fetch('https://www.reddit.com/r/memes/top.json?sort=top&t=day&limit=500')
+            .then(res => res.json());
+
+        const raw = children.filter((c: any) => c.data.over_18 === false);
+        const meme = raw[Math.floor(Math.random() * raw.length)].data;
```

Comment:
This seems like a potential bug to me. What happens if for whatever reason, the filter makes this empty ie all of over 18. You would have `undefined.data` which would throw an error

Reply:
You are definitely correct. I have resolved this.